packages/jobs/lib/execution/sync.ts

```diff
@@ -268,49 +269,45 @@ export async function handleSyncSuccess({ nangoProps }: { nangoProps: NangoProps
 
             syncPayload.records[model] = { added, updated, deleted };
 
-            const webhookSettings = await externalWebhookService.get(nangoProps.environmentId);
```

Comment:
moving the `externalWebhookService.get` outside of the models loop. No need to do it multiple times.